2024-10-15 13:46:35 [INFO][com.easypan.EasyPanApplication][logStarting][55]-> Starting EasyPanApplication using Java 1.8.0_422 on JIANGHAO with PID 18836 (D:\apache-maven-3.9.1\easypan\target\classes started by 蒋浩 in D:\apache-maven-3.9.1\easypan)
2024-10-15 13:46:35 [INFO][com.easypan.EasyPanApplication][logStartupProfileInfo][635]-> No active profile set, falling back to default profiles: default
2024-10-15 13:46:35 [INFO][org.springframework.data.repository.config.RepositoryConfigurationDelegate][multipleStoresDetected][262]-> Multiple Spring Data modules found, entering strict repository configuration mode!
2024-10-15 13:46:35 [INFO][org.springframework.data.repository.config.RepositoryConfigurationDelegate][registerRepositoriesIn][132]-> Bootstrapping Spring Data Redis repositories in DEFAULT mode.
2024-10-15 13:46:35 [INFO][org.springframework.data.repository.config.RepositoryConfigurationDelegate][registerRepositoriesIn][201]-> Finished Spring Data repository scanning in 8 ms. Found 0 Redis repository interfaces.
2024-10-15 13:46:36 [INFO][org.springframework.boot.web.embedded.tomcat.TomcatWebServer][initialize][108]-> Tomcat initialized with port(s): 8080 (http)
2024-10-15 13:46:36 [INFO][org.apache.coyote.http11.Http11NioProtocol][log][173]-> Initializing ProtocolHandler ["http-nio-8080"]
2024-10-15 13:46:36 [INFO][org.apache.catalina.core.StandardService][log][173]-> Starting service [Tomcat]
2024-10-15 13:46:36 [INFO][org.apache.catalina.core.StandardEngine][log][173]-> Starting Servlet engine: [Apache Tomcat/9.0.55]
2024-10-15 13:46:36 [INFO][org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/api]][log][173]-> Initializing Spring embedded WebApplicationContext
2024-10-15 13:46:36 [INFO][org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext][prepareWebApplicationContext][290]-> Root WebApplicationContext: initialization completed in 892 ms
2024-10-15 13:46:37 [INFO][org.apache.coyote.http11.Http11NioProtocol][log][173]-> Starting ProtocolHandler ["http-nio-8080"]
2024-10-15 13:46:37 [INFO][org.springframework.boot.web.embedded.tomcat.TomcatWebServer][start][220]-> Tomcat started on port(s): 8080 (http) with context path '/api'
2024-10-15 13:46:37 [INFO][com.easypan.EasyPanApplication][logStarted][61]-> Started EasyPanApplication in 2.238 seconds (JVM running for 3.215)
2024-10-15 13:46:37 [INFO][com.zaxxer.hikari.HikariDataSource][getConnection][110]-> HikariCPDatasource - Starting...
2024-10-15 13:46:37 [INFO][com.zaxxer.hikari.HikariDataSource][getConnection][123]-> HikariCPDatasource - Start completed.
2024-10-15 13:46:37 [ERROR][com.easypan.InitRun][run][30]-> 服务启动成功，可以开始愉快的开发了
2024-10-15 13:46:49 [INFO][org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/api]][log][173]-> Initializing Spring DispatcherServlet 'dispatcherServlet'
2024-10-15 13:46:49 [INFO][org.springframework.web.servlet.DispatcherServlet][initServletBean][525]-> Initializing Servlet 'dispatcherServlet'
2024-10-15 13:46:49 [INFO][org.springframework.web.servlet.DispatcherServlet][initServletBean][547]-> Completed initialization in 0 ms
2024-10-15 13:46:49 [WARN][org.apache.catalina.connector.Request][log][173]-> Creating the temporary upload location [C:\Users\蒋浩\AppData\Local\Temp\tomcat.8080.6412150764005839959\work\Tomcat\localhost\api\log\temp] as it is required by the servlet [dispatcherServlet]
2024-10-15 13:46:49 [ERROR][com.easypan.aspect.GlobalOperationAspect][interceptorDo][79]-> 全局拦截器异常
com.easypan.exception.BusinessException: 登录超时，请重新登录
2024-10-15 13:46:49 [ERROR][com.easypan.controller.AGlobalExceptionHandlerController][handleException][25]-> 请求错误，请求地址http://localhost:8080/api/file/loadDataList,错误信息:
com.easypan.exception.BusinessException: 登录超时，请重新登录
2024-10-15 13:49:37 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@3308c419 (The last packet successfully received from the server was 156,148 milliseconds ago. The last packet sent successfully to the server was 156,148 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 13:49:37 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@5a98f8cc (The last packet successfully received from the server was 179,920 milliseconds ago. The last packet sent successfully to the server was 179,920 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 13:49:37 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@2f45d78a (The last packet successfully received from the server was 179,917 milliseconds ago. The last packet sent successfully to the server was 179,917 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 13:49:37 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@3a125004 (The last packet successfully received from the server was 179,915 milliseconds ago. The last packet sent successfully to the server was 179,915 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 13:49:37 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@c6ee6f5 (The last packet successfully received from the server was 179,913 milliseconds ago. The last packet sent successfully to the server was 179,913 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 13:52:37 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@10d54d14 (The last packet successfully received from the server was 180,026 milliseconds ago. The last packet sent successfully to the server was 180,026 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 13:52:37 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@434daddc (The last packet successfully received from the server was 180,011 milliseconds ago. The last packet sent successfully to the server was 180,011 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 13:52:37 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@3b11c0c8 (The last packet successfully received from the server was 180,011 milliseconds ago. The last packet sent successfully to the server was 180,011 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 13:52:37 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@1968753 (The last packet successfully received from the server was 180,011 milliseconds ago. The last packet sent successfully to the server was 180,011 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 13:52:37 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@44d7fe76 (The last packet successfully received from the server was 180,018 milliseconds ago. The last packet sent successfully to the server was 180,018 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 13:58:37 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@2f19b693 (The last packet successfully received from the server was 180,014 milliseconds ago. The last packet sent successfully to the server was 180,015 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 13:58:37 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@57bd597e (The last packet successfully received from the server was 360,025 milliseconds ago. The last packet sent successfully to the server was 360,025 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 13:58:37 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@30898535 (The last packet successfully received from the server was 360,026 milliseconds ago. The last packet sent successfully to the server was 360,026 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 13:58:37 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@52461099 (The last packet successfully received from the server was 360,027 milliseconds ago. The last packet sent successfully to the server was 360,027 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 13:58:37 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@f21faf7 (The last packet successfully received from the server was 360,028 milliseconds ago. The last packet sent successfully to the server was 360,028 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:01:37 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@2c96bc16 (The last packet successfully received from the server was 180,015 milliseconds ago. The last packet sent successfully to the server was 180,015 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:01:37 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@4fcf407d (The last packet successfully received from the server was 180,015 milliseconds ago. The last packet sent successfully to the server was 180,015 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:01:37 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@34fea4b2 (The last packet successfully received from the server was 180,013 milliseconds ago. The last packet sent successfully to the server was 180,014 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:01:37 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@2d11c837 (The last packet successfully received from the server was 180,013 milliseconds ago. The last packet sent successfully to the server was 180,013 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:01:37 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@134c121 (The last packet successfully received from the server was 180,012 milliseconds ago. The last packet sent successfully to the server was 180,012 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:04:37 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@75d75cdd (The last packet successfully received from the server was 180,010 milliseconds ago. The last packet sent successfully to the server was 180,010 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:04:37 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@713e9c0c (The last packet successfully received from the server was 180,011 milliseconds ago. The last packet sent successfully to the server was 180,011 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:04:37 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@288444dc (The last packet successfully received from the server was 180,011 milliseconds ago. The last packet sent successfully to the server was 180,011 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:04:37 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@677ad4a5 (The last packet successfully received from the server was 180,011 milliseconds ago. The last packet sent successfully to the server was 180,011 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:04:37 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@123a1f18 (The last packet successfully received from the server was 180,010 milliseconds ago. The last packet sent successfully to the server was 180,010 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:07:37 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@1dc93b2b (The last packet successfully received from the server was 180,005 milliseconds ago. The last packet sent successfully to the server was 180,006 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:07:37 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@6e271398 (The last packet successfully received from the server was 180,006 milliseconds ago. The last packet sent successfully to the server was 180,006 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:07:37 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@7025c5a4 (The last packet successfully received from the server was 180,006 milliseconds ago. The last packet sent successfully to the server was 180,006 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:07:37 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@2461b8e8 (The last packet successfully received from the server was 180,005 milliseconds ago. The last packet sent successfully to the server was 180,005 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:07:37 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@308c86ec (The last packet successfully received from the server was 180,005 milliseconds ago. The last packet sent successfully to the server was 180,005 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:08:01 [INFO][com.zaxxer.hikari.HikariDataSource][close][350]-> HikariCPDatasource - Shutdown initiated...
2024-10-15 14:08:01 [INFO][com.zaxxer.hikari.HikariDataSource][close][352]-> HikariCPDatasource - Shutdown completed.
2024-10-15 14:08:08 [INFO][com.easypan.EasyPanApplication][logStarting][55]-> Starting EasyPanApplication using Java 1.8.0_422 on JIANGHAO with PID 6240 (D:\apache-maven-3.9.1\easypan\target\classes started by 蒋浩 in D:\apache-maven-3.9.1\easypan)
2024-10-15 14:08:08 [INFO][com.easypan.EasyPanApplication][logStartupProfileInfo][635]-> No active profile set, falling back to default profiles: default
2024-10-15 14:08:09 [INFO][org.springframework.data.repository.config.RepositoryConfigurationDelegate][multipleStoresDetected][262]-> Multiple Spring Data modules found, entering strict repository configuration mode!
2024-10-15 14:08:09 [INFO][org.springframework.data.repository.config.RepositoryConfigurationDelegate][registerRepositoriesIn][132]-> Bootstrapping Spring Data Redis repositories in DEFAULT mode.
2024-10-15 14:08:09 [INFO][org.springframework.data.repository.config.RepositoryConfigurationDelegate][registerRepositoriesIn][201]-> Finished Spring Data repository scanning in 8 ms. Found 0 Redis repository interfaces.
2024-10-15 14:08:09 [INFO][org.springframework.boot.web.embedded.tomcat.TomcatWebServer][initialize][108]-> Tomcat initialized with port(s): 8080 (http)
2024-10-15 14:08:09 [INFO][org.apache.coyote.http11.Http11NioProtocol][log][173]-> Initializing ProtocolHandler ["http-nio-8080"]
2024-10-15 14:08:09 [INFO][org.apache.catalina.core.StandardService][log][173]-> Starting service [Tomcat]
2024-10-15 14:08:09 [INFO][org.apache.catalina.core.StandardEngine][log][173]-> Starting Servlet engine: [Apache Tomcat/9.0.55]
2024-10-15 14:08:09 [INFO][org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/api]][log][173]-> Initializing Spring embedded WebApplicationContext
2024-10-15 14:08:09 [INFO][org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext][prepareWebApplicationContext][290]-> Root WebApplicationContext: initialization completed in 847 ms
2024-10-15 14:08:10 [INFO][org.apache.coyote.http11.Http11NioProtocol][log][173]-> Starting ProtocolHandler ["http-nio-8080"]
2024-10-15 14:08:10 [INFO][org.springframework.boot.web.embedded.tomcat.TomcatWebServer][start][220]-> Tomcat started on port(s): 8080 (http) with context path '/api'
2024-10-15 14:08:10 [INFO][com.easypan.EasyPanApplication][logStarted][61]-> Started EasyPanApplication in 2.096 seconds (JVM running for 2.796)
2024-10-15 14:08:10 [INFO][com.zaxxer.hikari.HikariDataSource][getConnection][110]-> HikariCPDatasource - Starting...
2024-10-15 14:08:10 [INFO][com.zaxxer.hikari.HikariDataSource][getConnection][123]-> HikariCPDatasource - Start completed.
2024-10-15 14:08:11 [ERROR][com.easypan.InitRun][run][30]-> 服务启动成功，可以开始愉快的开发了
2024-10-15 14:08:15 [INFO][org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/api]][log][173]-> Initializing Spring DispatcherServlet 'dispatcherServlet'
2024-10-15 14:08:15 [INFO][org.springframework.web.servlet.DispatcherServlet][initServletBean][525]-> Initializing Servlet 'dispatcherServlet'
2024-10-15 14:08:15 [INFO][org.springframework.web.servlet.DispatcherServlet][initServletBean][547]-> Completed initialization in 1 ms
2024-10-15 14:08:15 [WARN][org.apache.catalina.connector.Request][log][173]-> Creating the temporary upload location [C:\Users\蒋浩\AppData\Local\Temp\tomcat.8080.7208977129126865661\work\Tomcat\localhost\api\log\temp] as it is required by the servlet [dispatcherServlet]
2024-10-15 14:08:15 [WARN][org.apache.catalina.connector.Request][log][173]-> Creating the temporary upload location [C:\Users\蒋浩\AppData\Local\Temp\tomcat.8080.7208977129126865661\work\Tomcat\localhost\api\log\temp] as it is required by the servlet [dispatcherServlet]
2024-10-15 14:08:15 [WARN][org.apache.catalina.connector.Request][log][173]-> Failed to create the upload location [C:\Users\蒋浩\AppData\Local\Temp\tomcat.8080.7208977129126865661\work\Tomcat\localhost\api\log\temp]
2024-10-15 14:08:15 [ERROR][com.easypan.aspect.GlobalOperationAspect][interceptorDo][79]-> 全局拦截器异常
com.easypan.exception.BusinessException: 登录超时，请重新登录
2024-10-15 14:08:15 [ERROR][com.easypan.controller.AGlobalExceptionHandlerController][handleException][25]-> 请求错误，请求地址http://localhost:8080/api/getUseSpace,错误信息:
com.easypan.exception.BusinessException: 登录超时，请重新登录
2024-10-15 14:08:15 [ERROR][com.easypan.aspect.GlobalOperationAspect][interceptorDo][79]-> 全局拦截器异常
com.easypan.exception.BusinessException: 登录超时，请重新登录
2024-10-15 14:08:15 [ERROR][com.easypan.controller.AGlobalExceptionHandlerController][handleException][25]-> 请求错误，请求地址http://localhost:8080/api/file/loadDataList,错误信息:
com.easypan.exception.BusinessException: 登录超时，请重新登录
2024-10-15 14:09:02 [INFO][com.zaxxer.hikari.HikariDataSource][close][350]-> HikariCPDatasource - Shutdown initiated...
2024-10-15 14:09:02 [INFO][com.zaxxer.hikari.HikariDataSource][close][352]-> HikariCPDatasource - Shutdown completed.
2024-10-15 14:09:07 [INFO][com.easypan.EasyPanApplication][logStarting][55]-> Starting EasyPanApplication using Java 1.8.0_422 on JIANGHAO with PID 5508 (D:\apache-maven-3.9.1\easypan\target\classes started by 蒋浩 in D:\apache-maven-3.9.1\easypan)
2024-10-15 14:09:07 [INFO][com.easypan.EasyPanApplication][logStartupProfileInfo][635]-> No active profile set, falling back to default profiles: default
2024-10-15 14:09:07 [INFO][org.springframework.data.repository.config.RepositoryConfigurationDelegate][multipleStoresDetected][262]-> Multiple Spring Data modules found, entering strict repository configuration mode!
2024-10-15 14:09:07 [INFO][org.springframework.data.repository.config.RepositoryConfigurationDelegate][registerRepositoriesIn][132]-> Bootstrapping Spring Data Redis repositories in DEFAULT mode.
2024-10-15 14:09:07 [INFO][org.springframework.data.repository.config.RepositoryConfigurationDelegate][registerRepositoriesIn][201]-> Finished Spring Data repository scanning in 11 ms. Found 0 Redis repository interfaces.
2024-10-15 14:09:08 [INFO][org.springframework.boot.web.embedded.tomcat.TomcatWebServer][initialize][108]-> Tomcat initialized with port(s): 8080 (http)
2024-10-15 14:09:08 [INFO][org.apache.coyote.http11.Http11NioProtocol][log][173]-> Initializing ProtocolHandler ["http-nio-8080"]
2024-10-15 14:09:08 [INFO][org.apache.catalina.core.StandardService][log][173]-> Starting service [Tomcat]
2024-10-15 14:09:08 [INFO][org.apache.catalina.core.StandardEngine][log][173]-> Starting Servlet engine: [Apache Tomcat/9.0.55]
2024-10-15 14:09:08 [INFO][org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/api]][log][173]-> Initializing Spring embedded WebApplicationContext
2024-10-15 14:09:08 [INFO][org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext][prepareWebApplicationContext][290]-> Root WebApplicationContext: initialization completed in 844 ms
2024-10-15 14:09:09 [INFO][org.apache.coyote.http11.Http11NioProtocol][log][173]-> Starting ProtocolHandler ["http-nio-8080"]
2024-10-15 14:09:09 [INFO][org.springframework.boot.web.embedded.tomcat.TomcatWebServer][start][220]-> Tomcat started on port(s): 8080 (http) with context path '/api'
2024-10-15 14:09:09 [INFO][com.easypan.EasyPanApplication][logStarted][61]-> Started EasyPanApplication in 1.907 seconds (JVM running for 2.613)
2024-10-15 14:09:09 [INFO][com.zaxxer.hikari.HikariDataSource][getConnection][110]-> HikariCPDatasource - Starting...
2024-10-15 14:09:09 [INFO][com.zaxxer.hikari.HikariDataSource][getConnection][123]-> HikariCPDatasource - Start completed.
2024-10-15 14:09:09 [ERROR][com.easypan.InitRun][run][30]-> 服务启动成功，可以开始愉快的开发了
2024-10-15 14:09:17 [INFO][org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/api]][log][173]-> Initializing Spring DispatcherServlet 'dispatcherServlet'
2024-10-15 14:09:17 [INFO][org.springframework.web.servlet.DispatcherServlet][initServletBean][525]-> Initializing Servlet 'dispatcherServlet'
2024-10-15 14:09:17 [INFO][org.springframework.web.servlet.DispatcherServlet][initServletBean][547]-> Completed initialization in 1 ms
2024-10-15 14:09:44 [INFO][com.zaxxer.hikari.HikariDataSource][close][350]-> HikariCPDatasource - Shutdown initiated...
2024-10-15 14:09:44 [INFO][com.zaxxer.hikari.HikariDataSource][close][352]-> HikariCPDatasource - Shutdown completed.
2024-10-15 14:09:48 [INFO][com.easypan.EasyPanApplication][logStarting][55]-> Starting EasyPanApplication using Java 1.8.0_422 on JIANGHAO with PID 11468 (D:\apache-maven-3.9.1\easypan\target\classes started by 蒋浩 in D:\apache-maven-3.9.1\easypan)
2024-10-15 14:09:48 [INFO][com.easypan.EasyPanApplication][logStartupProfileInfo][635]-> No active profile set, falling back to default profiles: default
2024-10-15 14:09:49 [INFO][org.springframework.data.repository.config.RepositoryConfigurationDelegate][multipleStoresDetected][262]-> Multiple Spring Data modules found, entering strict repository configuration mode!
2024-10-15 14:09:49 [INFO][org.springframework.data.repository.config.RepositoryConfigurationDelegate][registerRepositoriesIn][132]-> Bootstrapping Spring Data Redis repositories in DEFAULT mode.
2024-10-15 14:09:49 [INFO][org.springframework.data.repository.config.RepositoryConfigurationDelegate][registerRepositoriesIn][201]-> Finished Spring Data repository scanning in 8 ms. Found 0 Redis repository interfaces.
2024-10-15 14:09:49 [INFO][org.springframework.boot.web.embedded.tomcat.TomcatWebServer][initialize][108]-> Tomcat initialized with port(s): 8080 (http)
2024-10-15 14:09:49 [INFO][org.apache.coyote.http11.Http11NioProtocol][log][173]-> Initializing ProtocolHandler ["http-nio-8080"]
2024-10-15 14:09:49 [INFO][org.apache.catalina.core.StandardService][log][173]-> Starting service [Tomcat]
2024-10-15 14:09:49 [INFO][org.apache.catalina.core.StandardEngine][log][173]-> Starting Servlet engine: [Apache Tomcat/9.0.55]
2024-10-15 14:09:49 [INFO][org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/api]][log][173]-> Initializing Spring embedded WebApplicationContext
2024-10-15 14:09:49 [INFO][org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext][prepareWebApplicationContext][290]-> Root WebApplicationContext: initialization completed in 860 ms
2024-10-15 14:09:50 [INFO][org.apache.coyote.http11.Http11NioProtocol][log][173]-> Starting ProtocolHandler ["http-nio-8080"]
2024-10-15 14:09:50 [INFO][org.springframework.boot.web.embedded.tomcat.TomcatWebServer][start][220]-> Tomcat started on port(s): 8080 (http) with context path '/api'
2024-10-15 14:09:50 [INFO][com.easypan.EasyPanApplication][logStarted][61]-> Started EasyPanApplication in 1.935 seconds (JVM running for 2.614)
2024-10-15 14:09:50 [INFO][com.zaxxer.hikari.HikariDataSource][getConnection][110]-> HikariCPDatasource - Starting...
2024-10-15 14:09:50 [INFO][com.zaxxer.hikari.HikariDataSource][getConnection][123]-> HikariCPDatasource - Start completed.
2024-10-15 14:09:51 [ERROR][com.easypan.InitRun][run][30]-> 服务启动成功，可以开始愉快的开发了
2024-10-15 14:09:54 [INFO][org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/api]][log][173]-> Initializing Spring DispatcherServlet 'dispatcherServlet'
2024-10-15 14:09:54 [INFO][org.springframework.web.servlet.DispatcherServlet][initServletBean][525]-> Initializing Servlet 'dispatcherServlet'
2024-10-15 14:09:54 [INFO][org.springframework.web.servlet.DispatcherServlet][initServletBean][547]-> Completed initialization in 1 ms
2024-10-15 14:11:15 [INFO][com.zaxxer.hikari.HikariDataSource][close][350]-> HikariCPDatasource - Shutdown initiated...
2024-10-15 14:11:15 [INFO][com.zaxxer.hikari.HikariDataSource][close][352]-> HikariCPDatasource - Shutdown completed.
2024-10-15 14:11:21 [INFO][com.easypan.EasyPanApplication][logStarting][55]-> Starting EasyPanApplication using Java 1.8.0_422 on JIANGHAO with PID 8268 (D:\apache-maven-3.9.1\easypan\target\classes started by 蒋浩 in D:\apache-maven-3.9.1\easypan)
2024-10-15 14:11:21 [INFO][com.easypan.EasyPanApplication][logStartupProfileInfo][635]-> No active profile set, falling back to default profiles: default
2024-10-15 14:11:21 [INFO][org.springframework.data.repository.config.RepositoryConfigurationDelegate][multipleStoresDetected][262]-> Multiple Spring Data modules found, entering strict repository configuration mode!
2024-10-15 14:11:21 [INFO][org.springframework.data.repository.config.RepositoryConfigurationDelegate][registerRepositoriesIn][132]-> Bootstrapping Spring Data Redis repositories in DEFAULT mode.
2024-10-15 14:11:21 [INFO][org.springframework.data.repository.config.RepositoryConfigurationDelegate][registerRepositoriesIn][201]-> Finished Spring Data repository scanning in 7 ms. Found 0 Redis repository interfaces.
2024-10-15 14:11:22 [INFO][org.springframework.boot.web.embedded.tomcat.TomcatWebServer][initialize][108]-> Tomcat initialized with port(s): 8080 (http)
2024-10-15 14:11:22 [INFO][org.apache.coyote.http11.Http11NioProtocol][log][173]-> Initializing ProtocolHandler ["http-nio-8080"]
2024-10-15 14:11:22 [INFO][org.apache.catalina.core.StandardService][log][173]-> Starting service [Tomcat]
2024-10-15 14:11:22 [INFO][org.apache.catalina.core.StandardEngine][log][173]-> Starting Servlet engine: [Apache Tomcat/9.0.55]
2024-10-15 14:11:22 [INFO][org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/api]][log][173]-> Initializing Spring embedded WebApplicationContext
2024-10-15 14:11:22 [INFO][org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext][prepareWebApplicationContext][290]-> Root WebApplicationContext: initialization completed in 847 ms
2024-10-15 14:11:22 [INFO][org.apache.coyote.http11.Http11NioProtocol][log][173]-> Starting ProtocolHandler ["http-nio-8080"]
2024-10-15 14:11:22 [INFO][org.springframework.boot.web.embedded.tomcat.TomcatWebServer][start][220]-> Tomcat started on port(s): 8080 (http) with context path '/api'
2024-10-15 14:11:22 [INFO][com.easypan.EasyPanApplication][logStarted][61]-> Started EasyPanApplication in 1.939 seconds (JVM running for 2.825)
2024-10-15 14:11:22 [INFO][com.zaxxer.hikari.HikariDataSource][getConnection][110]-> HikariCPDatasource - Starting...
2024-10-15 14:11:23 [INFO][com.zaxxer.hikari.HikariDataSource][getConnection][123]-> HikariCPDatasource - Start completed.
2024-10-15 14:11:23 [ERROR][com.easypan.InitRun][run][30]-> 服务启动成功，可以开始愉快的开发了
2024-10-15 14:11:27 [INFO][org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/api]][log][173]-> Initializing Spring DispatcherServlet 'dispatcherServlet'
2024-10-15 14:11:27 [INFO][org.springframework.web.servlet.DispatcherServlet][initServletBean][525]-> Initializing Servlet 'dispatcherServlet'
2024-10-15 14:11:27 [INFO][org.springframework.web.servlet.DispatcherServlet][initServletBean][547]-> Completed initialization in 2 ms
2024-10-15 14:11:42 [WARN][org.apache.catalina.connector.Request][log][173]-> Creating the temporary upload location [C:\Users\蒋浩\AppData\Local\Temp\tomcat.8080.5591055071762224621\work\Tomcat\localhost\api\log\temp] as it is required by the servlet [dispatcherServlet]
2024-10-15 14:11:42 [ERROR][com.easypan.controller.AGlobalExceptionHandlerController][handleException][25]-> 请求错误，请求地址http://localhost:8080/api/login,错误信息:
com.easypan.exception.BusinessException: 图片验证码不正确
2024-10-15 14:11:51 [ERROR][com.easypan.controller.AGlobalExceptionHandlerController][handleException][25]-> 请求错误，请求地址http://localhost:8080/api/login,错误信息:
com.easypan.exception.BusinessException: 图片验证码不正确
2024-10-15 14:14:23 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@36c36072 (The last packet successfully received from the server was 141,202 milliseconds ago. The last packet sent successfully to the server was 141,202 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:14:23 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@76bfa641 (The last packet successfully received from the server was 179,922 milliseconds ago. The last packet sent successfully to the server was 179,922 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:14:23 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@5b528814 (The last packet successfully received from the server was 179,920 milliseconds ago. The last packet sent successfully to the server was 179,920 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:14:23 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@2cfa7a72 (The last packet successfully received from the server was 179,917 milliseconds ago. The last packet sent successfully to the server was 179,917 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:14:23 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@5970256a (The last packet successfully received from the server was 179,915 milliseconds ago. The last packet sent successfully to the server was 179,915 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:24:57 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@1dab3dc9 (The last packet successfully received from the server was 634,551 milliseconds ago. The last packet sent successfully to the server was 634,551 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:24:57 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@3a7ecfb4 (The last packet successfully received from the server was 634,549 milliseconds ago. The last packet sent successfully to the server was 634,549 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:24:57 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@58079615 (The last packet successfully received from the server was 634,548 milliseconds ago. The last packet sent successfully to the server was 634,548 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:24:57 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@1c86ada9 (The last packet successfully received from the server was 634,546 milliseconds ago. The last packet sent successfully to the server was 634,546 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:24:57 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@11196110 (The last packet successfully received from the server was 634,545 milliseconds ago. The last packet sent successfully to the server was 634,545 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:24:57 [WARN][com.zaxxer.hikari.pool.HikariPool][run][788]-> HikariCPDatasource - Thread starvation or clock leap detected (housekeeper delta=8m4s557ms52µs).
2024-10-15 14:25:53 [INFO][com.zaxxer.hikari.HikariDataSource][close][350]-> HikariCPDatasource - Shutdown initiated...
2024-10-15 14:25:53 [INFO][com.zaxxer.hikari.HikariDataSource][close][352]-> HikariCPDatasource - Shutdown completed.
2024-10-15 14:25:57 [INFO][com.easypan.EasyPanApplication][logStarting][55]-> Starting EasyPanApplication using Java 1.8.0_422 on JIANGHAO with PID 15724 (D:\apache-maven-3.9.1\easypan\target\classes started by 蒋浩 in D:\apache-maven-3.9.1\easypan)
2024-10-15 14:25:57 [INFO][com.easypan.EasyPanApplication][logStartupProfileInfo][635]-> No active profile set, falling back to default profiles: default
2024-10-15 14:25:58 [INFO][org.springframework.data.repository.config.RepositoryConfigurationDelegate][multipleStoresDetected][262]-> Multiple Spring Data modules found, entering strict repository configuration mode!
2024-10-15 14:25:58 [INFO][org.springframework.data.repository.config.RepositoryConfigurationDelegate][registerRepositoriesIn][132]-> Bootstrapping Spring Data Redis repositories in DEFAULT mode.
2024-10-15 14:25:58 [INFO][org.springframework.data.repository.config.RepositoryConfigurationDelegate][registerRepositoriesIn][201]-> Finished Spring Data repository scanning in 7 ms. Found 0 Redis repository interfaces.
2024-10-15 14:25:58 [INFO][org.springframework.boot.web.embedded.tomcat.TomcatWebServer][initialize][108]-> Tomcat initialized with port(s): 8080 (http)
2024-10-15 14:25:58 [INFO][org.apache.coyote.http11.Http11NioProtocol][log][173]-> Initializing ProtocolHandler ["http-nio-8080"]
2024-10-15 14:25:58 [INFO][org.apache.catalina.core.StandardService][log][173]-> Starting service [Tomcat]
2024-10-15 14:25:58 [INFO][org.apache.catalina.core.StandardEngine][log][173]-> Starting Servlet engine: [Apache Tomcat/9.0.55]
2024-10-15 14:25:58 [INFO][org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/api]][log][173]-> Initializing Spring embedded WebApplicationContext
2024-10-15 14:25:58 [INFO][org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext][prepareWebApplicationContext][290]-> Root WebApplicationContext: initialization completed in 826 ms
2024-10-15 14:25:59 [INFO][org.apache.coyote.http11.Http11NioProtocol][log][173]-> Starting ProtocolHandler ["http-nio-8080"]
2024-10-15 14:25:59 [INFO][org.springframework.boot.web.embedded.tomcat.TomcatWebServer][start][220]-> Tomcat started on port(s): 8080 (http) with context path '/api'
2024-10-15 14:25:59 [INFO][com.easypan.EasyPanApplication][logStarted][61]-> Started EasyPanApplication in 1.854 seconds (JVM running for 2.574)
2024-10-15 14:25:59 [INFO][com.zaxxer.hikari.HikariDataSource][getConnection][110]-> HikariCPDatasource - Starting...
2024-10-15 14:25:59 [INFO][com.zaxxer.hikari.HikariDataSource][getConnection][123]-> HikariCPDatasource - Start completed.
2024-10-15 14:26:00 [ERROR][com.easypan.InitRun][run][30]-> 服务启动成功，可以开始愉快的开发了
2024-10-15 14:28:59 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@2c02afc0 (The last packet successfully received from the server was 180,021 milliseconds ago. The last packet sent successfully to the server was 180,022 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:28:59 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@8f6286 (The last packet successfully received from the server was 179,926 milliseconds ago. The last packet sent successfully to the server was 179,927 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:28:59 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@39caa1f8 (The last packet successfully received from the server was 179,923 milliseconds ago. The last packet sent successfully to the server was 179,923 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:28:59 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@72e2f52a (The last packet successfully received from the server was 179,921 milliseconds ago. The last packet sent successfully to the server was 179,921 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:28:59 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@7fe66b07 (The last packet successfully received from the server was 179,919 milliseconds ago. The last packet sent successfully to the server was 179,919 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:31:42 [INFO][org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/api]][log][173]-> Initializing Spring DispatcherServlet 'dispatcherServlet'
2024-10-15 14:31:42 [INFO][org.springframework.web.servlet.DispatcherServlet][initServletBean][525]-> Initializing Servlet 'dispatcherServlet'
2024-10-15 14:31:42 [INFO][org.springframework.web.servlet.DispatcherServlet][initServletBean][547]-> Completed initialization in 2 ms
2024-10-15 14:31:59 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@68af1926 (The last packet successfully received from the server was 180,014 milliseconds ago. The last packet sent successfully to the server was 180,014 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:31:59 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@41f30648 (The last packet successfully received from the server was 180,014 milliseconds ago. The last packet sent successfully to the server was 180,014 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:31:59 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@70ad248c (The last packet successfully received from the server was 180,008 milliseconds ago. The last packet sent successfully to the server was 180,008 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:31:59 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@60db6596 (The last packet successfully received from the server was 180,006 milliseconds ago. The last packet sent successfully to the server was 180,006 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:31:59 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@628da72b (The last packet successfully received from the server was 180,005 milliseconds ago. The last packet sent successfully to the server was 180,005 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:34:59 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@2285e6de (The last packet successfully received from the server was 180,011 milliseconds ago. The last packet sent successfully to the server was 180,011 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:34:59 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@16f29820 (The last packet successfully received from the server was 180,011 milliseconds ago. The last packet sent successfully to the server was 180,011 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:34:59 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@416e016 (The last packet successfully received from the server was 180,010 milliseconds ago. The last packet sent successfully to the server was 180,010 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:34:59 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@928555c (The last packet successfully received from the server was 180,009 milliseconds ago. The last packet sent successfully to the server was 180,009 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 14:34:59 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@5e30a30c (The last packet successfully received from the server was 180,008 milliseconds ago. The last packet sent successfully to the server was 180,008 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:08:49 [WARN][com.zaxxer.hikari.pool.HikariPool][run][788]-> HikariCPDatasource - Thread starvation or clock leap detected (housekeeper delta=32m19s309ms597µs100ns).
2024-10-15 15:12:48 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@664ca7d9 (The last packet successfully received from the server was 180,007 milliseconds ago. The last packet sent successfully to the server was 180,009 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:12:48 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@22815814 (The last packet successfully received from the server was 239,783 milliseconds ago. The last packet sent successfully to the server was 239,784 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:12:48 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@99536e8 (The last packet successfully received from the server was 239,782 milliseconds ago. The last packet sent successfully to the server was 239,782 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:12:48 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@a643a6c (The last packet successfully received from the server was 239,781 milliseconds ago. The last packet sent successfully to the server was 239,781 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:12:48 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@79a4efd2 (The last packet successfully received from the server was 239,780 milliseconds ago. The last packet sent successfully to the server was 239,780 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:15:48 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@714f17bd (The last packet successfully received from the server was 180,009 milliseconds ago. The last packet sent successfully to the server was 180,010 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:15:48 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@4d75a55 (The last packet successfully received from the server was 180,008 milliseconds ago. The last packet sent successfully to the server was 180,008 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:15:48 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@3fe956dc (The last packet successfully received from the server was 180,005 milliseconds ago. The last packet sent successfully to the server was 180,005 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:15:48 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@36124c63 (The last packet successfully received from the server was 180,008 milliseconds ago. The last packet sent successfully to the server was 180,008 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:15:48 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@73a1c8e7 (The last packet successfully received from the server was 180,006 milliseconds ago. The last packet sent successfully to the server was 180,006 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:18:18 [WARN][org.apache.catalina.connector.Request][log][173]-> Creating the temporary upload location [C:\Users\蒋浩\AppData\Local\Temp\tomcat.8080.3918301055117378175\work\Tomcat\localhost\api\log\temp] as it is required by the servlet [dispatcherServlet]
2024-10-15 15:18:18 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@1763a720 (The last packet successfully received from the server was 150,046 milliseconds ago. The last packet sent successfully to the server was 150,047 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:18:18 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@582575d (The last packet successfully received from the server was 150,047 milliseconds ago. The last packet sent successfully to the server was 150,047 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:18:18 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@8582235 (The last packet successfully received from the server was 150,048 milliseconds ago. The last packet sent successfully to the server was 150,048 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:18:18 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@288ceb9 (The last packet successfully received from the server was 150,049 milliseconds ago. The last packet sent successfully to the server was 150,049 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:18:18 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@4d8841f6 (The last packet successfully received from the server was 150,050 milliseconds ago. The last packet sent successfully to the server was 150,050 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:18:18 [ERROR][com.easypan.controller.AGlobalExceptionHandlerController][handleException][25]-> 请求错误，请求地址http://localhost:8080/api/sendEmailCode,错误信息:
com.easypan.exception.BusinessException: 邮箱已经存在
2024-10-15 15:21:48 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@7d7178dd (The last packet successfully received from the server was 180,007 milliseconds ago. The last packet sent successfully to the server was 180,008 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:21:48 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@70380186 (The last packet successfully received from the server was 209,968 milliseconds ago. The last packet sent successfully to the server was 209,968 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:21:48 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@66e20d18 (The last packet successfully received from the server was 209,967 milliseconds ago. The last packet sent successfully to the server was 209,967 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:21:48 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@10a33689 (The last packet successfully received from the server was 209,966 milliseconds ago. The last packet sent successfully to the server was 209,966 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:21:48 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@501a04f3 (The last packet successfully received from the server was 209,965 milliseconds ago. The last packet sent successfully to the server was 209,965 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:24:48 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@16624091 (The last packet successfully received from the server was 180,011 milliseconds ago. The last packet sent successfully to the server was 180,012 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:24:48 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@1f41892a (The last packet successfully received from the server was 180,012 milliseconds ago. The last packet sent successfully to the server was 180,012 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:24:48 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@f956da6 (The last packet successfully received from the server was 180,011 milliseconds ago. The last packet sent successfully to the server was 180,011 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:24:48 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@bf7c20d (The last packet successfully received from the server was 180,011 milliseconds ago. The last packet sent successfully to the server was 180,011 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:24:48 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@752899a2 (The last packet successfully received from the server was 180,010 milliseconds ago. The last packet sent successfully to the server was 180,010 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:27:48 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@426fefc9 (The last packet successfully received from the server was 180,003 milliseconds ago. The last packet sent successfully to the server was 180,004 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:27:48 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@4669d8ef (The last packet successfully received from the server was 180,003 milliseconds ago. The last packet sent successfully to the server was 180,003 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:27:48 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@3a91a220 (The last packet successfully received from the server was 180,003 milliseconds ago. The last packet sent successfully to the server was 180,003 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:27:48 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@7bd131c (The last packet successfully received from the server was 180,002 milliseconds ago. The last packet sent successfully to the server was 180,002 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:27:48 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@1dcbcd7a (The last packet successfully received from the server was 180,001 milliseconds ago. The last packet sent successfully to the server was 180,001 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:30:48 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@225ae0a3 (The last packet successfully received from the server was 180,008 milliseconds ago. The last packet sent successfully to the server was 180,009 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:30:48 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@e7396d2 (The last packet successfully received from the server was 180,008 milliseconds ago. The last packet sent successfully to the server was 180,008 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:30:48 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@75615e30 (The last packet successfully received from the server was 180,008 milliseconds ago. The last packet sent successfully to the server was 180,008 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:30:48 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@4bf7c7 (The last packet successfully received from the server was 180,007 milliseconds ago. The last packet sent successfully to the server was 180,007 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:30:48 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@4c076dd3 (The last packet successfully received from the server was 180,007 milliseconds ago. The last packet sent successfully to the server was 180,007 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:33:48 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@21bff2a3 (The last packet successfully received from the server was 180,008 milliseconds ago. The last packet sent successfully to the server was 180,008 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:33:48 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@98b5f78 (The last packet successfully received from the server was 180,008 milliseconds ago. The last packet sent successfully to the server was 180,008 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:33:48 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@52bc2d41 (The last packet successfully received from the server was 180,007 milliseconds ago. The last packet sent successfully to the server was 180,008 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:33:48 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@c48e75f (The last packet successfully received from the server was 180,007 milliseconds ago. The last packet sent successfully to the server was 180,007 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:33:48 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@5ce9ab66 (The last packet successfully received from the server was 180,006 milliseconds ago. The last packet sent successfully to the server was 180,006 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 15:36:45 [INFO][com.zaxxer.hikari.HikariDataSource][close][350]-> HikariCPDatasource - Shutdown initiated...
2024-10-15 15:36:45 [INFO][com.zaxxer.hikari.HikariDataSource][close][352]-> HikariCPDatasource - Shutdown completed.
2024-10-15 16:21:48 [INFO][com.easypan.EasyPanApplication][logStarting][55]-> Starting EasyPanApplication using Java 1.8.0_422 on JIANGHAO with PID 3012 (D:\apache-maven-3.9.1\easypan\target\classes started by 蒋浩 in D:\apache-maven-3.9.1\easypan)
2024-10-15 16:21:48 [INFO][com.easypan.EasyPanApplication][logStartupProfileInfo][635]-> No active profile set, falling back to default profiles: default
2024-10-15 16:21:49 [INFO][org.springframework.data.repository.config.RepositoryConfigurationDelegate][multipleStoresDetected][262]-> Multiple Spring Data modules found, entering strict repository configuration mode!
2024-10-15 16:21:49 [INFO][org.springframework.data.repository.config.RepositoryConfigurationDelegate][registerRepositoriesIn][132]-> Bootstrapping Spring Data Redis repositories in DEFAULT mode.
2024-10-15 16:21:49 [INFO][org.springframework.data.repository.config.RepositoryConfigurationDelegate][registerRepositoriesIn][201]-> Finished Spring Data repository scanning in 10 ms. Found 0 Redis repository interfaces.
2024-10-15 16:21:49 [INFO][org.springframework.boot.web.embedded.tomcat.TomcatWebServer][initialize][108]-> Tomcat initialized with port(s): 8080 (http)
2024-10-15 16:21:49 [INFO][org.apache.coyote.http11.Http11NioProtocol][log][173]-> Initializing ProtocolHandler ["http-nio-8080"]
2024-10-15 16:21:49 [INFO][org.apache.catalina.core.StandardService][log][173]-> Starting service [Tomcat]
2024-10-15 16:21:49 [INFO][org.apache.catalina.core.StandardEngine][log][173]-> Starting Servlet engine: [Apache Tomcat/9.0.55]
2024-10-15 16:21:49 [INFO][org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/api]][log][173]-> Initializing Spring embedded WebApplicationContext
2024-10-15 16:21:49 [INFO][org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext][prepareWebApplicationContext][290]-> Root WebApplicationContext: initialization completed in 1191 ms
2024-10-15 16:21:50 [INFO][org.apache.coyote.http11.Http11NioProtocol][log][173]-> Starting ProtocolHandler ["http-nio-8080"]
2024-10-15 16:21:50 [INFO][org.springframework.boot.web.embedded.tomcat.TomcatWebServer][start][220]-> Tomcat started on port(s): 8080 (http) with context path '/api'
2024-10-15 16:21:50 [INFO][com.easypan.EasyPanApplication][logStarted][61]-> Started EasyPanApplication in 2.759 seconds (JVM running for 3.593)
2024-10-15 16:21:50 [INFO][com.zaxxer.hikari.HikariDataSource][getConnection][110]-> HikariCPDatasource - Starting...
2024-10-15 16:21:51 [INFO][com.zaxxer.hikari.HikariDataSource][getConnection][123]-> HikariCPDatasource - Start completed.
2024-10-15 16:21:51 [ERROR][com.easypan.InitRun][run][30]-> 服务启动成功，可以开始愉快的开发了
2024-10-15 16:21:56 [INFO][org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/api]][log][173]-> Initializing Spring DispatcherServlet 'dispatcherServlet'
2024-10-15 16:21:56 [INFO][org.springframework.web.servlet.DispatcherServlet][initServletBean][525]-> Initializing Servlet 'dispatcherServlet'
2024-10-15 16:21:56 [INFO][org.springframework.web.servlet.DispatcherServlet][initServletBean][547]-> Completed initialization in 1 ms
2024-10-15 16:24:51 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@37e6df38 (The last packet successfully received from the server was 180,026 milliseconds ago. The last packet sent successfully to the server was 180,033 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:24:51 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@5e57a7b6 (The last packet successfully received from the server was 179,930 milliseconds ago. The last packet sent successfully to the server was 179,930 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:24:51 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@5f49bc5b (The last packet successfully received from the server was 179,927 milliseconds ago. The last packet sent successfully to the server was 179,927 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:24:51 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@234a85d0 (The last packet successfully received from the server was 179,925 milliseconds ago. The last packet sent successfully to the server was 179,925 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:24:51 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@5319a556 (The last packet successfully received from the server was 179,938 milliseconds ago. The last packet sent successfully to the server was 179,938 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:27:51 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@22291ba9 (The last packet successfully received from the server was 180,010 milliseconds ago. The last packet sent successfully to the server was 180,010 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:27:51 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@4b1742cd (The last packet successfully received from the server was 180,010 milliseconds ago. The last packet sent successfully to the server was 180,010 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:27:51 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@7d25ad49 (The last packet successfully received from the server was 180,010 milliseconds ago. The last packet sent successfully to the server was 180,010 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:27:51 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@173b0517 (The last packet successfully received from the server was 180,010 milliseconds ago. The last packet sent successfully to the server was 180,010 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:27:51 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@11767a6f (The last packet successfully received from the server was 180,010 milliseconds ago. The last packet sent successfully to the server was 180,010 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:30:51 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@3019f50a (The last packet successfully received from the server was 180,017 milliseconds ago. The last packet sent successfully to the server was 180,017 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:30:51 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@5d42f526 (The last packet successfully received from the server was 180,019 milliseconds ago. The last packet sent successfully to the server was 180,019 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:30:51 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@f8b164b (The last packet successfully received from the server was 180,005 milliseconds ago. The last packet sent successfully to the server was 180,005 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:30:51 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@70c51315 (The last packet successfully received from the server was 180,006 milliseconds ago. The last packet sent successfully to the server was 180,006 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:30:51 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@2538f0eb (The last packet successfully received from the server was 180,007 milliseconds ago. The last packet sent successfully to the server was 180,007 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:33:51 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@60f251f9 (The last packet successfully received from the server was 180,019 milliseconds ago. The last packet sent successfully to the server was 180,020 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:33:51 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@234234fc (The last packet successfully received from the server was 180,019 milliseconds ago. The last packet sent successfully to the server was 180,019 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:33:51 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@26b2ad25 (The last packet successfully received from the server was 180,017 milliseconds ago. The last packet sent successfully to the server was 180,017 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:33:51 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@543d4d93 (The last packet successfully received from the server was 180,016 milliseconds ago. The last packet sent successfully to the server was 180,016 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:33:51 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@2c3fc7da (The last packet successfully received from the server was 180,015 milliseconds ago. The last packet sent successfully to the server was 180,016 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:35:13 [INFO][com.zaxxer.hikari.HikariDataSource][close][350]-> HikariCPDatasource - Shutdown initiated...
2024-10-15 16:35:13 [INFO][com.zaxxer.hikari.HikariDataSource][close][352]-> HikariCPDatasource - Shutdown completed.
2024-10-15 16:35:22 [INFO][com.easypan.EasyPanApplication][logStarting][55]-> Starting EasyPanApplication using Java 1.8.0_422 on JIANGHAO with PID 5516 (D:\apache-maven-3.9.1\easypan\target\classes started by 蒋浩 in D:\apache-maven-3.9.1\easypan)
2024-10-15 16:35:22 [INFO][com.easypan.EasyPanApplication][logStartupProfileInfo][635]-> No active profile set, falling back to default profiles: default
2024-10-15 16:35:23 [INFO][org.springframework.data.repository.config.RepositoryConfigurationDelegate][multipleStoresDetected][262]-> Multiple Spring Data modules found, entering strict repository configuration mode!
2024-10-15 16:35:23 [INFO][org.springframework.data.repository.config.RepositoryConfigurationDelegate][registerRepositoriesIn][132]-> Bootstrapping Spring Data Redis repositories in DEFAULT mode.
2024-10-15 16:35:23 [INFO][org.springframework.data.repository.config.RepositoryConfigurationDelegate][registerRepositoriesIn][201]-> Finished Spring Data repository scanning in 12 ms. Found 0 Redis repository interfaces.
2024-10-15 16:35:23 [INFO][org.springframework.boot.web.embedded.tomcat.TomcatWebServer][initialize][108]-> Tomcat initialized with port(s): 8080 (http)
2024-10-15 16:35:23 [INFO][org.apache.coyote.http11.Http11NioProtocol][log][173]-> Initializing ProtocolHandler ["http-nio-8080"]
2024-10-15 16:35:23 [INFO][org.apache.catalina.core.StandardService][log][173]-> Starting service [Tomcat]
2024-10-15 16:35:23 [INFO][org.apache.catalina.core.StandardEngine][log][173]-> Starting Servlet engine: [Apache Tomcat/9.0.55]
2024-10-15 16:35:23 [INFO][org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/api]][log][173]-> Initializing Spring embedded WebApplicationContext
2024-10-15 16:35:23 [INFO][org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext][prepareWebApplicationContext][290]-> Root WebApplicationContext: initialization completed in 854 ms
2024-10-15 16:35:24 [INFO][org.apache.coyote.http11.Http11NioProtocol][log][173]-> Starting ProtocolHandler ["http-nio-8080"]
2024-10-15 16:35:24 [INFO][org.springframework.boot.web.embedded.tomcat.TomcatWebServer][start][220]-> Tomcat started on port(s): 8080 (http) with context path '/api'
2024-10-15 16:35:24 [INFO][com.easypan.EasyPanApplication][logStarted][61]-> Started EasyPanApplication in 1.933 seconds (JVM running for 2.614)
2024-10-15 16:35:24 [INFO][com.zaxxer.hikari.HikariDataSource][getConnection][110]-> HikariCPDatasource - Starting...
2024-10-15 16:35:24 [INFO][com.zaxxer.hikari.HikariDataSource][getConnection][123]-> HikariCPDatasource - Start completed.
2024-10-15 16:35:25 [ERROR][com.easypan.InitRun][run][30]-> 服务启动成功，可以开始愉快的开发了
2024-10-15 16:35:37 [INFO][org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/api]][log][173]-> Initializing Spring DispatcherServlet 'dispatcherServlet'
2024-10-15 16:35:37 [INFO][org.springframework.web.servlet.DispatcherServlet][initServletBean][525]-> Initializing Servlet 'dispatcherServlet'
2024-10-15 16:35:37 [INFO][org.springframework.web.servlet.DispatcherServlet][initServletBean][547]-> Completed initialization in 0 ms
2024-10-15 16:36:34 [WARN][org.apache.catalina.connector.Request][log][173]-> Creating the temporary upload location [C:\Users\蒋浩\AppData\Local\Temp\tomcat.8080.3667570565391401609\work\Tomcat\localhost\api\log\temp] as it is required by the servlet [dispatcherServlet]
2024-10-15 16:36:34 [ERROR][com.easypan.controller.AGlobalExceptionHandlerController][handleException][25]-> 请求错误，请求地址http://localhost:8080/api/sendEmailCode,错误信息:
com.easypan.exception.BusinessException: 邮箱已经存在
2024-10-15 16:41:24 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@107d7c5c (The last packet successfully received from the server was 180,014 milliseconds ago. The last packet sent successfully to the server was 180,015 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:41:24 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@579bf637 (The last packet successfully received from the server was 359,942 milliseconds ago. The last packet sent successfully to the server was 359,942 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:41:24 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@2c7fff04 (The last packet successfully received from the server was 359,939 milliseconds ago. The last packet sent successfully to the server was 359,939 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:41:24 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@3c2ccd49 (The last packet successfully received from the server was 359,937 milliseconds ago. The last packet sent successfully to the server was 359,937 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:41:24 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@5153f3c4 (The last packet successfully received from the server was 359,934 milliseconds ago. The last packet sent successfully to the server was 359,934 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:44:24 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@aa5de72 (The last packet successfully received from the server was 180,009 milliseconds ago. The last packet sent successfully to the server was 180,009 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:44:24 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@1e3a08ea (The last packet successfully received from the server was 180,005 milliseconds ago. The last packet sent successfully to the server was 180,006 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:44:24 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@4b72b38f (The last packet successfully received from the server was 180,005 milliseconds ago. The last packet sent successfully to the server was 180,005 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:44:24 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@6f80a2dd (The last packet successfully received from the server was 180,004 milliseconds ago. The last packet sent successfully to the server was 180,004 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:44:24 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@77463d9b (The last packet successfully received from the server was 180,003 milliseconds ago. The last packet sent successfully to the server was 180,003 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:47:24 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@674534ab (The last packet successfully received from the server was 180,008 milliseconds ago. The last packet sent successfully to the server was 180,009 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:47:24 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@4b55ba0d (The last packet successfully received from the server was 180,009 milliseconds ago. The last packet sent successfully to the server was 180,009 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:47:24 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@5a07fea8 (The last packet successfully received from the server was 180,008 milliseconds ago. The last packet sent successfully to the server was 180,008 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:47:24 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@14a9792f (The last packet successfully received from the server was 180,007 milliseconds ago. The last packet sent successfully to the server was 180,007 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:47:24 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@56f4202a (The last packet successfully received from the server was 180,006 milliseconds ago. The last packet sent successfully to the server was 180,006 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:50:24 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@7d3dc8bc (The last packet successfully received from the server was 180,009 milliseconds ago. The last packet sent successfully to the server was 180,010 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:50:24 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@22a5f4ac (The last packet successfully received from the server was 180,008 milliseconds ago. The last packet sent successfully to the server was 180,008 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:50:24 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@520f6e0d (The last packet successfully received from the server was 180,006 milliseconds ago. The last packet sent successfully to the server was 180,006 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:50:24 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@41e424d8 (The last packet successfully received from the server was 180,004 milliseconds ago. The last packet sent successfully to the server was 180,004 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:50:24 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@737eb0c5 (The last packet successfully received from the server was 180,001 milliseconds ago. The last packet sent successfully to the server was 180,001 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:52:45 [ERROR][com.easypan.controller.AGlobalExceptionHandlerController][handleException][25]-> 请求错误，请求地址http://localhost:8080/api/login,错误信息:
com.easypan.exception.BusinessException: 图片验证码不正确
2024-10-15 16:52:54 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@6bc421df (The last packet successfully received from the server was 149,491 milliseconds ago. The last packet sent successfully to the server was 149,491 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:52:54 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@3199280b (The last packet successfully received from the server was 149,491 milliseconds ago. The last packet sent successfully to the server was 149,491 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:52:54 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@1a0538ec (The last packet successfully received from the server was 149,507 milliseconds ago. The last packet sent successfully to the server was 149,507 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:52:54 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@500528aa (The last packet successfully received from the server was 149,507 milliseconds ago. The last packet sent successfully to the server was 149,507 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:52:54 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@5c6e0a6 (The last packet successfully received from the server was 149,496 milliseconds ago. The last packet sent successfully to the server was 149,496 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:56:24 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@31f90632 (The last packet successfully received from the server was 180,013 milliseconds ago. The last packet sent successfully to the server was 180,013 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:56:24 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@2b0324d1 (The last packet successfully received from the server was 210,513 milliseconds ago. The last packet sent successfully to the server was 210,513 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:56:24 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@278c2ab6 (The last packet successfully received from the server was 210,514 milliseconds ago. The last packet sent successfully to the server was 210,514 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:56:24 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@5e3d60e3 (The last packet successfully received from the server was 210,515 milliseconds ago. The last packet sent successfully to the server was 210,515 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:56:24 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@40730e24 (The last packet successfully received from the server was 210,516 milliseconds ago. The last packet sent successfully to the server was 210,516 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:58:33 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@2d5d5d79 (The last packet successfully received from the server was 129,082 milliseconds ago. The last packet sent successfully to the server was 129,082 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:58:33 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@7c30b964 (The last packet successfully received from the server was 129,080 milliseconds ago. The last packet sent successfully to the server was 129,080 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:58:33 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@5d947a8a (The last packet successfully received from the server was 129,094 milliseconds ago. The last packet sent successfully to the server was 129,094 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:58:33 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@1a6552d4 (The last packet successfully received from the server was 129,092 milliseconds ago. The last packet sent successfully to the server was 129,092 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 16:58:33 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@63858d2 (The last packet successfully received from the server was 129,090 milliseconds ago. The last packet sent successfully to the server was 129,090 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 17:02:24 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@59aeba17 (The last packet successfully received from the server was 180,015 milliseconds ago. The last packet sent successfully to the server was 180,015 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 17:02:24 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@65e60963 (The last packet successfully received from the server was 230,920 milliseconds ago. The last packet sent successfully to the server was 230,920 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 17:02:24 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@40064e62 (The last packet successfully received from the server was 230,920 milliseconds ago. The last packet sent successfully to the server was 230,920 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 17:02:24 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@6cd42a77 (The last packet successfully received from the server was 230,920 milliseconds ago. The last packet sent successfully to the server was 230,920 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 17:02:24 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@574b0bb7 (The last packet successfully received from the server was 230,904 milliseconds ago. The last packet sent successfully to the server was 230,904 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 17:08:58 [WARN][com.zaxxer.hikari.pool.HikariPool][run][788]-> HikariCPDatasource - Thread starvation or clock leap detected (housekeeper delta=6m33s493ms132µs500ns).
2024-10-15 17:11:28 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@70b48656 (The last packet successfully received from the server was 543,482 milliseconds ago. The last packet sent successfully to the server was 543,482 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 17:11:28 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@32672f1e (The last packet successfully received from the server was 543,484 milliseconds ago. The last packet sent successfully to the server was 543,484 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 17:11:28 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@2da2aeb3 (The last packet successfully received from the server was 543,485 milliseconds ago. The last packet sent successfully to the server was 543,485 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 17:11:28 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@79701b67 (The last packet successfully received from the server was 543,470 milliseconds ago. The last packet sent successfully to the server was 543,470 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 17:11:28 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@302a2249 (The last packet successfully received from the server was 543,471 milliseconds ago. The last packet sent successfully to the server was 543,471 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 17:14:28 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@351127f (The last packet successfully received from the server was 180,008 milliseconds ago. The last packet sent successfully to the server was 180,009 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 17:14:28 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@1bdad732 (The last packet successfully received from the server was 180,010 milliseconds ago. The last packet sent successfully to the server was 180,010 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 17:14:28 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@570c5258 (The last packet successfully received from the server was 180,008 milliseconds ago. The last packet sent successfully to the server was 180,008 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 17:14:28 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@5c45602c (The last packet successfully received from the server was 180,006 milliseconds ago. The last packet sent successfully to the server was 180,007 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 17:14:28 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@3a72a8e7 (The last packet successfully received from the server was 180,004 milliseconds ago. The last packet sent successfully to the server was 180,004 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 17:17:28 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@42f799ed (The last packet successfully received from the server was 180,021 milliseconds ago. The last packet sent successfully to the server was 180,021 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 17:17:28 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@1ade7650 (The last packet successfully received from the server was 180,025 milliseconds ago. The last packet sent successfully to the server was 180,025 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 17:17:28 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@94f1cac (The last packet successfully received from the server was 180,020 milliseconds ago. The last packet sent successfully to the server was 180,020 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 17:17:28 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@7709d499 (The last packet successfully received from the server was 180,020 milliseconds ago. The last packet sent successfully to the server was 180,020 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 17:17:28 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@64720fda (The last packet successfully received from the server was 180,017 milliseconds ago. The last packet sent successfully to the server was 180,017 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 17:20:28 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@7cac4e8c (The last packet successfully received from the server was 180,010 milliseconds ago. The last packet sent successfully to the server was 180,010 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 17:20:28 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@31402b3e (The last packet successfully received from the server was 180,010 milliseconds ago. The last packet sent successfully to the server was 180,010 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 17:20:28 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@491cfd (The last packet successfully received from the server was 180,005 milliseconds ago. The last packet sent successfully to the server was 180,005 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 17:20:28 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@1e0ffa56 (The last packet successfully received from the server was 180,008 milliseconds ago. The last packet sent successfully to the server was 180,008 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 17:20:28 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@1c31c50b (The last packet successfully received from the server was 180,008 milliseconds ago. The last packet sent successfully to the server was 180,008 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 17:23:28 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@64fe4bf2 (The last packet successfully received from the server was 180,005 milliseconds ago. The last packet sent successfully to the server was 180,005 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 17:23:28 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@17c9544f (The last packet successfully received from the server was 180,008 milliseconds ago. The last packet sent successfully to the server was 180,008 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 17:23:28 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@7b1eb99a (The last packet successfully received from the server was 180,009 milliseconds ago. The last packet sent successfully to the server was 180,009 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 17:23:28 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@6d29dcd0 (The last packet successfully received from the server was 180,010 milliseconds ago. The last packet sent successfully to the server was 180,010 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 17:23:28 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@7efb485c (The last packet successfully received from the server was 180,004 milliseconds ago. The last packet sent successfully to the server was 180,004 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 17:25:42 [INFO][com.zaxxer.hikari.HikariDataSource][close][350]-> HikariCPDatasource - Shutdown initiated...
2024-10-15 17:25:42 [INFO][com.zaxxer.hikari.HikariDataSource][close][352]-> HikariCPDatasource - Shutdown completed.
2024-10-15 17:25:46 [INFO][com.easypan.EasyPanApplication][logStarting][55]-> Starting EasyPanApplication using Java 1.8.0_422 on JIANGHAO with PID 16728 (D:\apache-maven-3.9.1\easypan\target\classes started by 蒋浩 in D:\apache-maven-3.9.1\easypan)
2024-10-15 17:25:46 [INFO][com.easypan.EasyPanApplication][logStartupProfileInfo][635]-> No active profile set, falling back to default profiles: default
2024-10-15 17:25:46 [INFO][org.springframework.data.repository.config.RepositoryConfigurationDelegate][multipleStoresDetected][262]-> Multiple Spring Data modules found, entering strict repository configuration mode!
2024-10-15 17:25:46 [INFO][org.springframework.data.repository.config.RepositoryConfigurationDelegate][registerRepositoriesIn][132]-> Bootstrapping Spring Data Redis repositories in DEFAULT mode.
2024-10-15 17:25:46 [INFO][org.springframework.data.repository.config.RepositoryConfigurationDelegate][registerRepositoriesIn][201]-> Finished Spring Data repository scanning in 9 ms. Found 0 Redis repository interfaces.
2024-10-15 17:25:47 [INFO][org.springframework.boot.web.embedded.tomcat.TomcatWebServer][initialize][108]-> Tomcat initialized with port(s): 8080 (http)
2024-10-15 17:25:47 [INFO][org.apache.coyote.http11.Http11NioProtocol][log][173]-> Initializing ProtocolHandler ["http-nio-8080"]
2024-10-15 17:25:47 [INFO][org.apache.catalina.core.StandardService][log][173]-> Starting service [Tomcat]
2024-10-15 17:25:47 [INFO][org.apache.catalina.core.StandardEngine][log][173]-> Starting Servlet engine: [Apache Tomcat/9.0.55]
2024-10-15 17:25:47 [INFO][org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/api]][log][173]-> Initializing Spring embedded WebApplicationContext
2024-10-15 17:25:47 [INFO][org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext][prepareWebApplicationContext][290]-> Root WebApplicationContext: initialization completed in 1072 ms
2024-10-15 17:25:49 [INFO][org.apache.coyote.http11.Http11NioProtocol][log][173]-> Starting ProtocolHandler ["http-nio-8080"]
2024-10-15 17:25:49 [INFO][org.springframework.boot.web.embedded.tomcat.TomcatWebServer][start][220]-> Tomcat started on port(s): 8080 (http) with context path '/api'
2024-10-15 17:25:49 [INFO][com.easypan.EasyPanApplication][logStarted][61]-> Started EasyPanApplication in 3.341 seconds (JVM running for 4.682)
2024-10-15 17:25:49 [INFO][com.zaxxer.hikari.HikariDataSource][getConnection][110]-> HikariCPDatasource - Starting...
2024-10-15 17:25:49 [INFO][com.zaxxer.hikari.HikariDataSource][getConnection][123]-> HikariCPDatasource - Start completed.
2024-10-15 17:25:50 [ERROR][com.easypan.InitRun][run][30]-> 服务启动成功，可以开始愉快的开发了
2024-10-15 17:27:07 [INFO][org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/api]][log][173]-> Initializing Spring DispatcherServlet 'dispatcherServlet'
2024-10-15 17:27:07 [INFO][org.springframework.web.servlet.DispatcherServlet][initServletBean][525]-> Initializing Servlet 'dispatcherServlet'
2024-10-15 17:27:07 [INFO][org.springframework.web.servlet.DispatcherServlet][initServletBean][547]-> Completed initialization in 1 ms
2024-10-15 17:27:07 [WARN][org.apache.catalina.connector.Request][log][173]-> Creating the temporary upload location [C:\Users\蒋浩\AppData\Local\Temp\tomcat.8080.4273801665306274175\work\Tomcat\localhost\api\log\temp] as it is required by the servlet [dispatcherServlet]
2024-10-15 17:27:07 [WARN][org.apache.catalina.connector.Request][log][173]-> Creating the temporary upload location [C:\Users\蒋浩\AppData\Local\Temp\tomcat.8080.4273801665306274175\work\Tomcat\localhost\api\log\temp] as it is required by the servlet [dispatcherServlet]
2024-10-15 17:27:07 [WARN][org.apache.catalina.connector.Request][log][173]-> Failed to create the upload location [C:\Users\蒋浩\AppData\Local\Temp\tomcat.8080.4273801665306274175\work\Tomcat\localhost\api\log\temp]
2024-10-15 17:27:07 [ERROR][com.easypan.aspect.GlobalOperationAspect][interceptorDo][79]-> 全局拦截器异常
com.easypan.exception.BusinessException: 登录超时，请重新登录
2024-10-15 17:27:07 [ERROR][com.easypan.controller.AGlobalExceptionHandlerController][handleException][25]-> 请求错误，请求地址http://localhost:8080/api/getUseSpace,错误信息:
com.easypan.exception.BusinessException: 登录超时，请重新登录
2024-10-15 17:27:07 [ERROR][com.easypan.aspect.GlobalOperationAspect][interceptorDo][79]-> 全局拦截器异常
com.easypan.exception.BusinessException: 登录超时，请重新登录
2024-10-15 17:27:07 [ERROR][com.easypan.controller.AGlobalExceptionHandlerController][handleException][25]-> 请求错误，请求地址http://localhost:8080/api/file/loadDataList,错误信息:
com.easypan.exception.BusinessException: 登录超时，请重新登录
2024-10-15 17:28:47 [WARN][com.zaxxer.hikari.pool.HikariPool][run][788]-> HikariCPDatasource - Thread starvation or clock leap detected (housekeeper delta=53s644ms769µs600ns).
2024-10-15 17:28:47 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@774bd4ac (The last packet successfully received from the server was 178,116 milliseconds ago. The last packet sent successfully to the server was 178,117 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 17:28:47 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@404ca0e (The last packet successfully received from the server was 178,069 milliseconds ago. The last packet sent successfully to the server was 178,069 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 17:28:47 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@2dc39096 (The last packet successfully received from the server was 178,070 milliseconds ago. The last packet sent successfully to the server was 178,070 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 17:28:47 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@2e58ee49 (The last packet successfully received from the server was 178,070 milliseconds ago. The last packet sent successfully to the server was 178,070 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 17:28:47 [WARN][com.zaxxer.hikari.pool.PoolBase][isConnectionAlive][184]-> HikariCPDatasource - Failed to validate connection com.mysql.cj.jdbc.ConnectionImpl@471d5d4a (The last packet successfully received from the server was 178,070 milliseconds ago. The last packet sent successfully to the server was 178,070 milliseconds ago. is longer than the server configured value of 'wait_timeout'. You should consider either expiring and/or testing connection validity before use in your application, increasing the server configured values for client timeouts, or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.). Possibly consider using a shorter maxLifetime value.
2024-10-15 17:31:04 [WARN][com.zaxxer.hikari.pool.HikariPool][run][788]-> HikariCPDatasource - Thread starvation or clock leap detected (housekeeper delta=1m11s613ms583µs500ns).
2024-10-15 17:31:08 [ERROR][com.easypan.controller.AGlobalExceptionHandlerController][handleException][25]-> 请求错误，请求地址http://localhost:8080/api/sendEmailCode,错误信息:
org.apache.catalina.connector.ClientAbortException: java.io.IOException: 你的主机中的软件中止了一个已建立的连接。
	at org.apache.catalina.connector.OutputBuffer.doFlush(OutputBuffer.java:310)
	at org.apache.catalina.connector.OutputBuffer.flush(OutputBuffer.java:273)
	at org.apache.catalina.connector.CoyoteOutputStream.flush(CoyoteOutputStream.java:118)
	at java.io.FilterOutputStream.flush(FilterOutputStream.java:140)
	at com.fasterxml.jackson.core.json.UTF8JsonGenerator.flush(UTF8JsonGenerator.java:1187)
	at com.fasterxml.jackson.databind.ObjectWriter.writeValue(ObjectWriter.java:1009)
	at org.springframework.http.converter.json.AbstractJackson2HttpMessageConverter.writeInternal(AbstractJackson2HttpMessageConverter.java:454)
	at org.springframework.http.converter.AbstractGenericHttpMessageConverter.write(AbstractGenericHttpMessageConverter.java:104)
	at org.springframework.web.servlet.mvc.method.annotation.AbstractMessageConverterMethodProcessor.writeWithMessageConverters(AbstractMessageConverterMethodProcessor.java:290)
	at org.springframework.web.servlet.mvc.method.annotation.RequestResponseBodyMethodProcessor.handleReturnValue(RequestResponseBodyMethodProcessor.java:183)
	at org.springframework.web.method.support.HandlerMethodReturnValueHandlerComposite.handleReturnValue(HandlerMethodReturnValueHandlerComposite.java:78)
	at org.springframework.web.servlet.mvc.method.annotation.ServletInvocableHandlerMethod.invokeAndHandle(ServletInvocableHandlerMethod.java:135)
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.invokeHandlerMethod(RequestMappingHandlerAdapter.java:895)
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.handleInternal(RequestMappingHandlerAdapter.java:808)
	at org.springframework.web.servlet.mvc.method.AbstractHandlerMethodAdapter.handle(AbstractHandlerMethodAdapter.java:87)
	at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:1067)
	at org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:963)
	at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:1006)
	at org.springframework.web.servlet.FrameworkServlet.doPost(FrameworkServlet.java:909)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:681)
	at org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:883)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.springframework.web.filter.RequestContextFilter.doFilterInternal(RequestContextFilter.java:100)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:119)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.springframework.web.filter.FormContentFilter.doFilterInternal(FormContentFilter.java:93)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:119)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:201)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:119)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:540)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:135)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:357)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:382)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:895)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1722)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
	at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:750)
Caused by: java.io.IOException: 你的主机中的软件中止了一个已建立的连接。
	at sun.nio.ch.SocketDispatcher.write0(Native Method)
	at sun.nio.ch.SocketDispatcher.write(SocketDispatcher.java:51)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:93)
	at sun.nio.ch.IOUtil.write(IOUtil.java:65)
	at sun.nio.ch.SocketChannelImpl.write(SocketChannelImpl.java:470)
	at org.apache.tomcat.util.net.NioChannel.write(NioChannel.java:135)
	at org.apache.tomcat.util.net.NioEndpoint$NioSocketWrapper.doWrite(NioEndpoint.java:1363)
	at org.apache.tomcat.util.net.SocketWrapperBase.doWrite(SocketWrapperBase.java:766)
	at org.apache.tomcat.util.net.SocketWrapperBase.flushBlocking(SocketWrapperBase.java:719)
	at org.apache.tomcat.util.net.SocketWrapperBase.flush(SocketWrapperBase.java:709)
	at org.apache.coyote.http11.Http11OutputBuffer$SocketOutputBuffer.flush(Http11OutputBuffer.java:573)
	at org.apache.coyote.http11.filters.IdentityOutputFilter.flush(IdentityOutputFilter.java:117)
	at org.apache.coyote.http11.Http11OutputBuffer.flush(Http11OutputBuffer.java:221)
	at org.apache.coyote.http11.Http11Processor.flush(Http11Processor.java:1205)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:402)
	at org.apache.coyote.Response.action(Response.java:209)
	at org.apache.catalina.connector.OutputBuffer.doFlush(OutputBuffer.java:306)
	... 54 common frames omitted
2024-10-15 17:31:08 [ERROR][com.easypan.controller.AGlobalExceptionHandlerController][handleException][25]-> 请求错误，请求地址http://localhost:8080/api/register,错误信息:
com.easypan.exception.BusinessException: 图片验证码不正确
2024-10-15 17:31:08 [WARN][org.springframework.web.servlet.mvc.method.annotation.ExceptionHandlerExceptionResolver][doResolveHandlerMethodException][434]-> Failure in @ExceptionHandler com.easypan.controller.AGlobalExceptionHandlerController#handleException(Exception, HttpServletRequest)
org.apache.catalina.connector.ClientAbortException: java.io.IOException: 你的主机中的软件中止了一个已建立的连接。
	at org.apache.catalina.connector.OutputBuffer.doFlush(OutputBuffer.java:310)
	at org.apache.catalina.connector.OutputBuffer.flush(OutputBuffer.java:273)
	at org.apache.catalina.connector.CoyoteOutputStream.flush(CoyoteOutputStream.java:118)
	at java.io.FilterOutputStream.flush(FilterOutputStream.java:140)
	at com.fasterxml.jackson.core.json.UTF8JsonGenerator.flush(UTF8JsonGenerator.java:1187)
	at com.fasterxml.jackson.databind.ObjectWriter.writeValue(ObjectWriter.java:1009)
	at org.springframework.http.converter.json.AbstractJackson2HttpMessageConverter.writeInternal(AbstractJackson2HttpMessageConverter.java:454)
	at org.springframework.http.converter.AbstractGenericHttpMessageConverter.write(AbstractGenericHttpMessageConverter.java:104)
	at org.springframework.web.servlet.mvc.method.annotation.AbstractMessageConverterMethodProcessor.writeWithMessageConverters(AbstractMessageConverterMethodProcessor.java:290)
	at org.springframework.web.servlet.mvc.method.annotation.RequestResponseBodyMethodProcessor.handleReturnValue(RequestResponseBodyMethodProcessor.java:183)
	at org.springframework.web.method.support.HandlerMethodReturnValueHandlerComposite.handleReturnValue(HandlerMethodReturnValueHandlerComposite.java:78)
	at org.springframework.web.servlet.mvc.method.annotation.ServletInvocableHandlerMethod.invokeAndHandle(ServletInvocableHandlerMethod.java:135)
	at org.springframework.web.servlet.mvc.method.annotation.ExceptionHandlerExceptionResolver.doResolveHandlerMethodException(ExceptionHandlerExceptionResolver.java:428)
	at org.springframework.web.servlet.handler.AbstractHandlerMethodExceptionResolver.doResolveException(AbstractHandlerMethodExceptionResolver.java:75)
	at org.springframework.web.servlet.handler.AbstractHandlerExceptionResolver.resolveException(AbstractHandlerExceptionResolver.java:142)
	at org.springframework.web.servlet.handler.HandlerExceptionResolverComposite.resolveException(HandlerExceptionResolverComposite.java:80)
	at org.springframework.web.servlet.DispatcherServlet.processHandlerException(DispatcherServlet.java:1327)
	at org.springframework.web.servlet.DispatcherServlet.processDispatchResult(DispatcherServlet.java:1138)
	at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:1084)
	at org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:963)
	at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:1006)
	at org.springframework.web.servlet.FrameworkServlet.doPost(FrameworkServlet.java:909)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:681)
	at org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:883)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.springframework.web.filter.RequestContextFilter.doFilterInternal(RequestContextFilter.java:100)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:119)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.springframework.web.filter.FormContentFilter.doFilterInternal(FormContentFilter.java:93)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:119)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:201)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:119)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:540)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:135)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:357)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:382)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:895)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1722)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
	at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:750)
Caused by: java.io.IOException: 你的主机中的软件中止了一个已建立的连接。
	at sun.nio.ch.SocketDispatcher.write0(Native Method)
	at sun.nio.ch.SocketDispatcher.write(SocketDispatcher.java:51)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:93)
	at sun.nio.ch.IOUtil.write(IOUtil.java:65)
	at sun.nio.ch.SocketChannelImpl.write(SocketChannelImpl.java:470)
	at org.apache.tomcat.util.net.NioChannel.write(NioChannel.java:135)
	at org.apache.tomcat.util.net.NioEndpoint$NioSocketWrapper.doWrite(NioEndpoint.java:1363)
	at org.apache.tomcat.util.net.SocketWrapperBase.doWrite(SocketWrapperBase.java:766)
	at org.apache.tomcat.util.net.SocketWrapperBase.flushBlocking(SocketWrapperBase.java:719)
	at org.apache.tomcat.util.net.SocketWrapperBase.flush(SocketWrapperBase.java:709)
	at org.apache.coyote.http11.Http11OutputBuffer$SocketOutputBuffer.flush(Http11OutputBuffer.java:573)
	at org.apache.coyote.http11.filters.IdentityOutputFilter.flush(IdentityOutputFilter.java:117)
	at org.apache.coyote.http11.Http11OutputBuffer.flush(Http11OutputBuffer.java:221)
	at org.apache.coyote.http11.Http11Processor.flush(Http11Processor.java:1205)
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:402)
	at org.apache.coyote.Response.action(Response.java:209)
	at org.apache.catalina.connector.OutputBuffer.doFlush(OutputBuffer.java:306)
	... 57 common frames omitted
2024-10-15 17:31:17 [INFO][com.zaxxer.hikari.HikariDataSource][close][350]-> HikariCPDatasource - Shutdown initiated...
2024-10-15 17:31:17 [INFO][com.zaxxer.hikari.HikariDataSource][close][352]-> HikariCPDatasource - Shutdown completed.
2024-10-15 17:49:03 [INFO][com.easypan.EasyPanApplication][logStarting][55]-> Starting EasyPanApplication using Java 1.8.0_422 on JIANGHAO with PID 8252 (D:\apache-maven-3.9.1\easypan\target\classes started by 蒋浩 in D:\apache-maven-3.9.1\easypan)
2024-10-15 17:49:03 [INFO][com.easypan.EasyPanApplication][logStartupProfileInfo][635]-> No active profile set, falling back to default profiles: default
2024-10-15 17:49:03 [INFO][org.springframework.data.repository.config.RepositoryConfigurationDelegate][multipleStoresDetected][262]-> Multiple Spring Data modules found, entering strict repository configuration mode!
2024-10-15 17:49:03 [INFO][org.springframework.data.repository.config.RepositoryConfigurationDelegate][registerRepositoriesIn][132]-> Bootstrapping Spring Data Redis repositories in DEFAULT mode.
2024-10-15 17:49:03 [INFO][org.springframework.data.repository.config.RepositoryConfigurationDelegate][registerRepositoriesIn][201]-> Finished Spring Data repository scanning in 13 ms. Found 0 Redis repository interfaces.
2024-10-15 17:49:03 [INFO][org.springframework.boot.web.embedded.tomcat.TomcatWebServer][initialize][108]-> Tomcat initialized with port(s): 8080 (http)
2024-10-15 17:49:03 [INFO][org.apache.coyote.http11.Http11NioProtocol][log][173]-> Initializing ProtocolHandler ["http-nio-8080"]
2024-10-15 17:49:03 [INFO][org.apache.catalina.core.StandardService][log][173]-> Starting service [Tomcat]
2024-10-15 17:49:03 [INFO][org.apache.catalina.core.StandardEngine][log][173]-> Starting Servlet engine: [Apache Tomcat/9.0.55]
2024-10-15 17:49:03 [INFO][org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/api]][log][173]-> Initializing Spring embedded WebApplicationContext
2024-10-15 17:49:03 [INFO][org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext][prepareWebApplicationContext][290]-> Root WebApplicationContext: initialization completed in 910 ms
2024-10-15 17:49:04 [INFO][org.apache.coyote.http11.Http11NioProtocol][log][173]-> Starting ProtocolHandler ["http-nio-8080"]
2024-10-15 17:49:04 [INFO][org.springframework.boot.web.embedded.tomcat.TomcatWebServer][start][220]-> Tomcat started on port(s): 8080 (http) with context path '/api'
2024-10-15 17:49:04 [INFO][com.easypan.EasyPanApplication][logStarted][61]-> Started EasyPanApplication in 2.032 seconds (JVM running for 2.729)
2024-10-15 17:49:04 [INFO][com.zaxxer.hikari.HikariDataSource][getConnection][110]-> HikariCPDatasource - Starting...
2024-10-15 17:49:04 [INFO][com.zaxxer.hikari.HikariDataSource][getConnection][123]-> HikariCPDatasource - Start completed.
2024-10-15 17:49:05 [ERROR][com.easypan.InitRun][run][30]-> 服务启动成功，可以开始愉快的开发了
2024-10-15 17:49:08 [INFO][org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/api]][log][173]-> Initializing Spring DispatcherServlet 'dispatcherServlet'
2024-10-15 17:49:08 [INFO][org.springframework.web.servlet.DispatcherServlet][initServletBean][525]-> Initializing Servlet 'dispatcherServlet'
2024-10-15 17:49:08 [INFO][org.springframework.web.servlet.DispatcherServlet][initServletBean][547]-> Completed initialization in 1 ms
2024-10-15 17:49:15 [WARN][org.apache.catalina.connector.Request][log][173]-> Creating the temporary upload location [C:\Users\蒋浩\AppData\Local\Temp\tomcat.8080.3522406972106478132\work\Tomcat\localhost\api\log\temp] as it is required by the servlet [dispatcherServlet]
2024-10-15 17:49:15 [ERROR][com.easypan.controller.AGlobalExceptionHandlerController][handleException][25]-> 请求错误，请求地址http://localhost:8080/api/login,错误信息:
com.easypan.exception.BusinessException: 图片验证码不正确
